server-spi/src/main/java/org/keycloak/storage/user/UserQueryProvider.java

```diff
@@ -405,17 +405,17 @@ default int getUsersCount(RealmModel realm, boolean includeServiceAccount) {
      * @return a non-null {@link Stream} of users that have the specified role.
      */
     default Stream<UserModel> getRoleMembersStream(RealmModel realm, RoleModel role) {
-        return Stream.empty();
+        return getRoleMembersStream(realm, role, null, null);
     }
 
     /**
      * Search for users that have a specific role with a specific roleId.
      *
+     * @param role
```

Comment:
nit: the javadoc's params are not aligned with actual method contract

Reply:
I will update all javadocs as part of Cleanup task. I believe it will be ready soon after merging this one. See https://issues.redhat.com/browse/KEYCLOAK-15524